shepherd/Timer.py

```diff
@@ -22,13 +68,47 @@ def __init__(self, timer_type, goal_name=None):
         self.goal_name = goal_name
 
     def start_timer(self, duration):
-        """Starts a new timer with the duration (seconds) and sets timer to active"""
-        pass
+        """Starts a new timer with the duration (seconds) and sets timer to active.
```

Comment:
I think this naming could get confusing, having `start_timer` also  be used to add additional time. Perhaps split this into a separate function, or rename it?